its/tests/src/test/java/its/tools/ItUtils.java

```diff
@@ -19,9 +19,72 @@
  */
 package its.tools;
 
+import java.io.IOException;
+import java.io.Reader;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Properties;
+
 public class ItUtils {
 
   public static final String SONAR_VERSION = getSonarVersion();
+  public static String javaVersion;
+  public static String pythonVersion;
+  public static String phpVersion;
+  public static String javascriptVersion;
+  public static String typescriptVersion;
+  public static String kotlinVersion;
+  public static String rubyVersion;
+  public static String scalaVersion;
+  public static String webVersion;
+  public static String xmlVersion;
+  public static String cobolVersion;
+  public static String apexVersion;
+  public static String tsqlVersion;
+  public static String cppVersion;
+
+  static {
+    if ("LATEST_RELEASE[6.7]".equals(System.getProperty("sonar.runtimeVersion"))) {
+      Properties props = new Properties();
+      try (Reader r = Files.newBufferedReader(Paths.get("../../core/src/main/resources/plugins_min_versions.txt"), StandardCharsets.UTF_8)) {
+        props.load(r);
+        javaVersion = props.getProperty("java");
+        pythonVersion = props.getProperty("python");
+        phpVersion = props.getProperty("php");
+        // Can't test the version 4.0.0 since there was a change of custom rule API
+        javascriptVersion = "4.2.0.6476";
+        typescriptVersion = props.getProperty("typescript");
+        kotlinVersion = props.getProperty("kotlin");
+        rubyVersion = props.getProperty("ruby");
+        scalaVersion = props.getProperty("sonarscala");
+        // Because of SONARHTML-91, it is simpler to test with 3.0.1.1444
```

Comment:
Is this still relevant?

Reply:
Yes, the minimal version we support is 3.0.0.1411. Another option would be to raise minimal version to 3.0.1.1444.